storybook/pages/AddEditSavedAddressPopupPage.qml

```diff
@@ -0,0 +1,160 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import SortFilterProxyModel 0.2
+
+import Storybook 1.0
+import Models 1.0
+import AppLayouts.Wallet.popups 1.0
+
+import utils 1.0
+
+SplitView {
+    orientation: Qt.Horizontal
+
+    Logs { id: logs }
+
+    PopupBackground {
+        id: popupBg
+
+        SplitView.fillWidth: true
+        SplitView.fillHeight: true
+
+        Button {
+            id: reopenButton
+            anchors.centerIn: parent
+            text: "Reopen"
+
+            onClicked: popupBg.openDialog()
+        }
+
+        function openDialog() {
+            popupComponent.createObject(popupBg)
+        }
+
+        Component.onCompleted: openDialog()
+
+        Component {
+            id: popupComponent
+            AddEditSavedAddressPopup {
+                visible: true
+                destroyOnClose: true
+                modal: false
+                closePolicy: Popup.NoAutoClose
+
+                flatNetworks: SortFilterProxyModel {
+                    sourceModel: NetworksModel.flatNetworks
+                    filters: ValueFilter { roleName: "isTest"; value: false }
+                }
+
+                store: QtObject {
```

Comment:
Store could be strongly typed in `AddEditSavedAddressPopup`.

Reply:
Ah, ok, I see, the store is singleton, so it requires bigger refactor here... Will be done as a separate works as there is ticket for that.

Reply:
Yeah, this is not my code so I admit I have no idea 🙂